.github/workflows/ci.yml

```diff
@@ -67,6 +67,11 @@ jobs:
             ninja-build \
             pkg-config
 
+      - name: Install uncrustify
+        run: |
+          brew update --quiet
```

Comment:
@gpanders mentioned this magic spell might help speed up homebrew:

```
export HOMEBREW_INSTALL_FROM_API=1
export HOMEBREW_NO_INSTALL_CLEANUP=1
export HOMEBREW_NO_AUTO_UPDATE=1
```

Reply:
Probably don't need `NO_AUTO_UPDATE` in this case, since you are already manually running `brew update` anyway.

I also am not 100% sure but if using `INSTALL_FROM_API` you *may* be able to omit `brew update` completely (which would shave off a lot of time).

Reply:
yeah in fact I seem to recall that homebrew really hates when CI runs `brew update` :)

Reply:
I have vague (I mean super vague) memory of `update` being required in some edge case, but I don't mind removing it and see if we encounter it.

Reply:
But with `INSTALL_FROM_API` maybe it's not needed as gpanders said. Let's try it!

Reply:
In that case you should keep `NO_AUTO_UPDATE` though.